jgrapht-io/src/main/java/org/jgrapht/io/SimpleGraphMLImporter.java

```diff
@@ -452,7 +452,7 @@ public void characters(char ch[], int start, int length)
             throws SAXException
         {
             if (insideData == 1) {
-                currentDataValue = new String(ch, start, length);
+                currentDataValue.append(new String(ch, start, length));
```

Comment:
Dunno whether it's faster, but there's a StringBuilder method append(CharSequence s, int start, int end) which allows you to skip the new String constructor.

Reply:
Yeah that's definitely the Java preferred way